src/ex4/server/db/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.json')[env];
```

Comment:
the code will be tidier of all "require"s would be on the top of the file. since in this case you need the "env" I would split it into 2 lines.
after line 5: `const configFile = require(__dirname + '/../config/config.json')`
at line 8: `const config = configFile[env]`